std/stream.d

```diff
@@ -3064,7 +3064,6 @@ class MmFileStream : TArrayStream!(MmFile) {
     if (isopen)
     {
       super.close();
-      delete buf;
```

Comment:
Please leave std.stream alone. The only fixes to be made on it are those that allow it to continue to compiler and work. It's deprecated.